faskinner.lic

```diff
@@ -0,0 +1,77 @@
+=begin
+Elanthipedia: https://elanthipedia.play.net/Lich_script_repository#faskinner
+
+# trainer noun should match exactly
+fa_skinning_trainer: crocodile 
+
+# Options:  First Aid, Skinning, Both or leave blank for whichever happens first
+fa_skinning_priority: Skinning 
+=end
+custom_require.call(%w[common events drinfomon equipmanager])
+
+class FaSkinner
+  include DRC
+  include DRCA
+
+  def initialize
+    @settings = get_settings
+    @trainer = @settings.fa_skinning_trainer
+    @priority = @settings.fa_skinning_priority
+    
+    EquipmentManager.new.empty_hands
+    get_fred
+
+    if @priority == 'First Aid' && DRSkill.getxp('First Aid') < 34
+      until DRSkill.getxp('First Aid') > 32
+        train_skills
+      end
+      echo ("Exiting because First Aid locked!")
+    elsif @priority == 'Skinning' && DRSkill.getxp('Skinning') < 34
+      until DRSkill.getxp('Skinning') > 32
+        train_skills
+      end
+      echo ("Exiting because Skinning locked!")
+    elsif @priority == 'Both' || @priority == 'both'
```

Comment:
I suggest something like `.downcase == 'both'` for all of these branches so you don't have to worry about casing.

Reply:
or `@priority =~ /both/i`. Although this is the only one that's case insensitive which fields weird (versus the other options, which I guess are proper skill names).